packages/react-components/recipes/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@fluentui/recipes",
+  "version": "0.0.0",
+  "private": true,
+  "description": "New fluentui react package",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "./dist/index.d.ts",
+  "sideEffects": false,
+  "files": [
+    "*.md",
+    "dist/*.d.ts",
+    "lib",
+    "lib-commonjs"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "generate-api": "just-scripts generate-api",
+    "lint": "just-scripts lint",
+    "test": "jest --passWithNoTests",
+    "type-check": "tsc -b tsconfig.json"
```

hmm this looks like you generated this before this landed to master https://github.com/microsoft/fluentui/pull/31142

with that I'm wondering what should we do about this particular library. it's gonna be solely for storybook modules and markdown content right ? no public api surface ?

```suggestion
    "type-check": "just-scripts type-check"
```

---

correct